backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/placement/PlacementService.java

```diff
@@ -9,53 +9,56 @@
 @Service
 public class PlacementService {
 
-    private final PlacementRepo placementRepo;
 
+    private final PlacementRepository placementRepository;
 
-    public PlacementService(PlacementRepo placementRepo) {
-        this.placementRepo = placementRepo;
+
+    public PlacementService(PlacementRepository placementRepository) {
+        this.placementRepository = placementRepository;
     }
 
     public List<Placement> getAllPlacements() {
-        return placementRepo.getAllPlacement();
+        return placementRepository.findAll();
     }
 
     public Placement addNewPlacement(NewPlacementData newRequestPlacement) {
 
         String newUuid = PlacementUtil.generateUuid();
+
         int placementNr = newRequestPlacement.placementNr();
         int totalSeats = newRequestPlacement.totalSeats();
 
         Placement newPlacement = new Placement(newUuid, placementNr, totalSeats);
-        placementRepo.addNewPlacement(newPlacement);
+        placementRepository.save(newPlacement);
 
         return newPlacement;
 
     }
 
     public Placement checkIfExist(String placementId) {
-        List<Placement> allPlacement = placementRepo.getAllPlacement();
-        Optional<Placement> placementToFind = allPlacement.stream()
-                .filter(placement -> placement.id().equals(placementId))
-                .findFirst();
+        Optional<Placement> placementToFind = placementRepository.findById(placementId);
 
         if (placementToFind.isEmpty()) {
             throw new IllegalArgumentException("Placement not Exist!");
         }
         return placementToFind.get();
     }
 
+
     public Placement updatePlacement(String placementId, Placement newData) {
-        Placement existPlacement = checkIfExist(placementId);
-        Placement updatedPlacement = new Placement(existPlacement.id(), existPlacement.placementNr(), newData.totalSeats());
+        Placement existPlacement = checkIfExist(placementId);// exist
 
-        placementRepo.updatePlacement(existPlacement, updatedPlacement);
+        Placement updatedPlacement = new Placement(existPlacement.id(), existPlacement.placementNr(), newData.totalSeats());
+        placementRepository.save(updatedPlacement);
         return updatedPlacement;
     }
 
     public boolean deletePlacement(String placementId) {
```

Comment:
delete should not return boolean

Reply:
A successful response SHOULD be 200 (OK) if the response includes an entity describing the status, 202 (Accepted) if the action has not yet been enacted, or 204 (No Content) if the action has been enacted but the response does not include an entity.

Reply:
We return true in delete service because we use it in unit test but we did not return  true in controller.
but we will change response to 204.